aten/src/ATen/native/quantized/TensorCompare.cpp

```diff
@@ -14,6 +14,19 @@ Tensor max_quantized_cpu(const Tensor& self) {
   return std::get<0>(self.reshape({-1}).max(/*dim=*/0));
 }
 
+Tensor& max_quantized_unary_out(const Tensor& self, Tensor& out) {
+  // First check if the devices match (CPU vs GPU)
```

Comment:
maybe add a comment just calling out that this kernel is implemented inefficiently for now (and could be improved).